api/proto/attest/attest.proto

```diff
@@ -3,7 +3,7 @@
 
 syntax = "proto3";
 
-import "common/common.proto";
+import "evecommon/evecommon.proto";
```

Comment:
Why can't evecommon be a Java-style package name to make sure we avoid collisions once and for all? IOW, what prevents you from calling it org.lfedge.eve.common  @kalyan-nidumolu ?